src/components/Root/FFetch.js

```diff
@@ -77,10 +77,27 @@ const OKAPI_FETCH_OPTIONS = {
 };
 
 export class FFetch {
-  constructor({ logger, store, rtrConfig }) {
```

Comment:
RTR config is exposed as part of the main `stripes-config`. Why should FFetch get passed it in a non-standard way?

Reply:
Strikes me as 'better testability' if you're able to just pass the config in vs having to mock it out/provide different mocks through some other way.